.github/workflows/update-css.yml

```
@@ -0,0 +1,89 @@
+
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: "Update CSS to version ${{ steps.fetch_release.outputs.tag }}"
+          branch: update-css-${{ steps.fetch_release.outputs.tag }}
+          title: "Update CSS to version ${{ steps.fetch_release.outputs.tag }}"
+          body: "This PR updates the CSS to version ${{ steps.fetch_release.outputs.tag }}."
+
+      - name: Wait for pull request to be merged
+        uses: actions/github-script@v3
```

actionlint (1.7.4)

the runner of "actions/github-script@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)